### PR TITLE
Qene records in BLorient9798

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created three records for two collections and one specimen from one of them following the example in https://github.com/BetaMasaheft/Works/pull/1150/files.

### CODE REVIEW

File: new/LIT7037Qene.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="dcterms:hasPart" active="LIT7037Qene" passive="LIT7038Qene"/>
+                </listRelation>
+                <div type="edition">
+                    <div type="textpart" subtype="incipit" xml:lang="gez">
+                        <note>The incipit is taken from <ref type="mss" corresp="BLorient9798"/> 
+                            as quoted by <bibl><ptr target="bm:Strelcyn1978BritishLibrary"/><citedRange unit="page">105</citedRange></bibl>.
+                           </note>
+                        <ab>ናሁ፡ እጽሕፍ፡ መዝገበ፡ ድርሳን፡ ዘይረስዎ፡ አዕረስዎ፡ አዕሩግ፡ የዋሃን፡ ወወራዙት፡ ማእምራን፡ ውስተ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡
```
Comment (karljonaskarlsson) [inline new/LIT7037Qene.xml:73]: አዕረስዎ፡ seems not to be in the catalogue

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: new/LIT7037Qene.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7037Qene.xml:73]: Thank you for your attentive reading!

**Review by karljonaskarlsson — APPROVED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7037Qene.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A collection of 90 <ref type="authFile" corresp="#Qene">qǝne songs</ref> of different kinds: 
+                    65 <foreign xml:lang="gez-tr">śǝllāse</foreign>, 
+                    12 <foreign xml:lang="gez-tr">mawaddǝs</foreign>, 
+                    8 <foreign xml:lang="gez-tr">wāzemā</foreign>,
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT7037Qene.xml:38]: I have seen it in records by Jonas. I do not know, whether it makes a difference.

File: new/LIT7038Qene.xml [hunk 3]
```diff
@@ -5,7 +5,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="en" xml:id="t1">Qǝne poem on ʾEwosṭātewos</title>
+                <title xml:lang="gez" xml:id="t1">እስመ፡ ወርኃ፡ ተድላ፡ ኮኖሙ፡ ለአዕባነ፡ ፍኖት፡ አፈ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎsma warḫā tadlā konomu la-ʾaʿǝbāna fǝnot ʾaf</title>
+                <title xml:lang="en" corresp="#t1">When its month is delight, there was a brim of the stones of the way</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7038Qene.xml:10]: Please add ... in transcription (followed by the indication Qǝne of ... type) and and transliteration, to show that the title is quotation.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7038Qene.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <fileDesc>
             <titleStmt>
                 <title xml:lang="gez" xml:id="t1">እስመ፡ ወርኃ፡ ተድላ፡ ኮኖሙ፡ ለአዕባነ፡ ፍኖት፡ አፈ፡</title>
-                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎsma warḫā tadlā konomu la-ʾaʿǝbāna fǝnot ʾaf</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎsma warḫā tadlā konomu la-ʾaʿǝbāna fǝnot ʾaf ... (Qǝne of śǝllāse-type)</title>
                 <title xml:lang="en" corresp="#t1">When its month is delight, there was a brim of the stones of the way</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT7038Qene.xml:10]: Sorry, in the translation os well, ...

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT7037Qene.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7037Qene.xml:40]: If we should not use gez-tr for transliteration, then maybe it should be removed from the possible values of xml:lang?

File: new/LIT7038Qene.xml [hunk 5]
```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7038Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እስመ፡ ወርኃ፡ ተድላ፡ ኮኖሙ፡ ለአዕባነ፡ ፍኖት፡ አፈ፡</title>
```
Comment (karljonaskarlsson) [inline new/LIT7038Qene.xml:8]: It seems like the second line is now used as the title instead of the first, or am I mistaken?

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

File: new/LIT7038Qene.xml (hunk 5 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7038Qene.xml:8]: I took the first line of which carries the rhyme, but I can also change or use both.

File: new/LIT7039QeneGabraMasih.xml [hunk 6]
```diff
@@ -5,7 +5,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader>
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">Collection of 9 qǝne poems attributed to Gabra Masiḥ</title>
+                <title xml:lang="gez" xml:id="t1">Collection of nine qǝne poems attributed to Gabra Masiḥ</title>
```
Comment (eu-genia) [inline new/LIT7039QeneGabraMasih.xml:8]: en

File: new/LIT7037Qene.xml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="#BLorient9798"/>
```
Comment (eu-genia) [inline new/LIT7037Qene.xml:27]: no `#` after corresp

File: new/LIT7037Qene.xml [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A collection of 90 <ref type="authFile" corresp="#Qene">qǝne poems</ref> of different kinds:
```
Comment (eu-genia) [inline new/LIT7037Qene.xml:37]: no `#` after corresp

`#` means a reference to an xml:id inside of the same record

File: new/LIT7037Qene.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/LIT7037Qene.xml:40]: gez-tr is ok

File: new/LIT7038Qene.xml [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="#BLorient9798"/>
```
Comment (eu-genia) [inline new/LIT7038Qene.xml:31]: remove `#`

File: new/LIT7039QeneGabraMasih.xml [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="#BLorient9798"/>
```
Comment (eu-genia) [inline new/LIT7039QeneGabraMasih.xml:25]: as above

File: new/LIT7039QeneGabraMasih.xml [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Nine <ref type="authFile" corresp="#Qene">qǝne songs</ref> attributed to person with the name
```
Comment (eu-genia) [inline new/LIT7039QeneGabraMasih.xml:35]: as above

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
is 7037 coming? not yet updated

File: new/LIT7037Qene.xml [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
```
Comment (eu-genia) [inline new/LIT7037Qene.xml:37]: this has not been corrected though marked as resolved

File: new/LIT7039QeneGabraMasih.xml [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Nine <ref type="authFile" corresp="Qene">qǝne songs</ref> attributed to person with the name
```
Comment (DenisNosnitsin1970) [inline new/LIT7039QeneGabraMasih.xml:35]: Please slightly reword the sentence, to make it more smooth ("nine qenes .... consisting of five qenes" seems a little odd to me). Qene are poems, not songs; ጉባዔ፡ ቃና፡ can be transcribed, and all terms encoded in a different way. I would included the manuscrpt in listRelations, as "attested in". I would also write out the numerals.

File: new/LIT7037Qene.xml (hunk 12 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT7037Qene.xml:37]: I checked all requests once again. I think, I have met all of them, now.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**